### PR TITLE
Fast forward ganache control

### PR DESCRIPTION
**Pull request by jFarms**
In one tab run 

> `yarn ganache`

In .evn.local set
> NEXT_PUBLIC_GANACHE_LOCAL=true

In a separate tab run
> yarn dev

Open wallet connect modal in rainbow kit and copy the URI.

To use the script to impersonate an address run

> node scripts/impersonate.js "put-uri-here"

The quotes around the uri are needed.

### CODE REVIEW

File: src/components/HeaderBar.tsx [hunk 1]
```diff
@@ -16,32 +16,56 @@ interface BlockSyncStatus {
...
 
+const SECONDS_PER_YEAR = 60 * 60 * 24 * 31 * 12;
+
+export const USE_GANACHE = process.env.NEXT_PUBLIC_GANACHE_LOCAL === 'true' && process.env.NODE_ENV === 'development';
+
 export const HeaderBar = (props: any) => {
   const [showResourcesModal, setShowResourcesModal] = React.useState<boolean>(false);
   const [syncStatus, setSyncStatus] = React.useState<BlockSyncStatus>();
   const {data: providerBlockNumber, refetch} = useBlockNumber();
+  const provider = useProvider() as any;
```
Comment (jFarms) [inline src/components/HeaderBar.tsx:27]: any type is required for `.send()`. Wagmi's provider type does not have this method even though it is defined as a method in `ethers.JsonRPCProvider`

**Comment by SolcAndMe** [comment]
Why not use port 8545?

**Comment by jFarms** [comment]
Update setup to use forked mainnet preferred

**Comment by jFarms** [comment]
Update Date.now usage to ensure interest calculations are correct

**Comment by jFarms** [comment]
Add snapshot and snapshot revert scripts

**Comment by jFarms** [comment]
Add optional block number to .env

**Comment by SolcAndMe** [comment]
closing in favor of #128

**Pull request closed without merging**